src/graphql/mutations/token.graphql

```diff
@@ -15,3 +15,9 @@ mutation DeleteColonyTokens($input: DeleteColonyTokensInput!) {
     id
   }
 }
+
+mutation CreateToken($input: CreateTokenInput!) {
+  createToken(input: $input) {
+    id
+  }
+}
```

Comment:
ah okay i missed the fact that this was calling an amplify function.

I have since updated the amplify function to introduce the concept of chains so that it knows what rpc url to call when checking for the token existence.

This still needs to be uploaded to the aws param store but you can test locally by applying this dirty patch that simulates what would happen if we were working live on chain

[safe-token.patch.gz](https://github.com/JoinColony/colonyCDapp/files/12463892/safe-token.patch.gz)

which adds these lines to the `fetchTokenFromChain` lambda:
```ts

  /* For dirty testing */
  switch (network) {
    case BINANCE_NETWORK.shortName:
      rpcURL = 'https://bsc.meowrpc.com';
      break;
    case ETHEREUM_NETWORK.shortName:
    default:
      rpcURL = 'https://eth.drpc.org';
  }
```

Importantly though, you have to apply this patch **after** you have created your colony as otherwise the setup process breaks as it cant find the token on the `ganache` chain